root/static/scripts/edit/MB/Control/Bubble.js

```diff
@@ -337,7 +337,12 @@ export default function initializeBubble(bubble, control, vm, canBeShown) {
   }
 
   ko.applyBindingsToNode($(bubble)[0], {bubble: bubbleDoc}, vm);
-  ko.applyBindingsToNode($(control)[0], {controlsBubble: bubbleDoc}, vm);
+
+  if (control) {
```

Comment:
I made this optional so the function can be used to initialize external link bubbles.